src/ms3/ms3-to-oas/merge-library-to-ms3.ts

```diff
@@ -0,0 +1,48 @@
+import * as MS3 from './../ms3-v1-api-interface';
+import { cloneDeep, forEach, map } from 'lodash';
+import { Library } from './../ms3-v1-library-interface';
+import { Trait } from './../ms3-v1-api-interface';
+
+class MergeLibraryToMs3 {
+  libraries: Library[];
+  constructor(private API: MS3.API) {}
+
+  checkDuplicates(targetArr: any, item: any, counter: number = 1): any {
+    const prop = item.name ? 'name' : 'title';
+    const traitNames = map(targetArr, (trait: any) => trait[prop]);
```

Comment:
why trait if this function is used for every api part? 
rename to smth appropriate